modules/docs/markdown/03-protocols/03-aws/01-aws.md

```diff
@@ -17,9 +17,15 @@ In `build.sbt`
 ```scala
 import smithy4s.codegen.BuildInfo._
 
+Compile / smithy4sAllowedNamespaces := List(
```

Comment:
Strange, I never had to set this key for AWS to work.

Reply:
Perhaps, because AWS's updated their schema definition:

https://github.com/aws/aws-sdk-js-v3/commits/41bb10dc093

https://github.com/aws/aws-sdk-js-v3/pull/4070

Reply:
Yeah I think we do. It should maybe be more specific like `smithy.api.*`

Reply:
do we even need to do so? With the `smithy4sGenerated` metadata I thiiiiiiink it might no longer be necessary to disable any namespaces by default.

For smithy4s-core, it contains this:

```smithy
$version: "2.0"

metadata smithy4sGenerated = [{smithy4sVersion: "0.17.0", namespaces: ["alloy", "smithy.waiters", "smithy.api"]}]
```

so I think it'd automatically ignore `smithy.api` and friends.

Reply:
I want to discourage people from using `smithy` prefixes to their namespaces in userland (people have actually tried that at $work).

Reply:
fair enough